test/dynamo/test_fallback.py

```diff
@@ -0,0 +1,78 @@
+import torch
+import torch_xla
+import torch_xla.core.xla_model as xm
+import torch._dynamo as dynamo
+import torch._dynamo.test_case as dt
+
+
+class XlaDynamoFallbackTest(dt.TestCase):
+
+  def test_operator_fallback(self):
+
+    def fn_fallback(M, mat1, mat2):
+      A = torch.cummin(M, 1)
+      B = torch.add(A[0], mat1)
+      return torch.add(B, mat2)
+
+    dynamo_fn = torch.compile(fn_fallback, backend="torchxla_trace_once")
+    M = torch.randn(5, 10)
+    mat1 = torch.randn(5, 10)
+    mat2 = torch.randn(5, 10)
+    xla_M = M.to(xm.xla_device())
+    xla_mat1 = mat1.to(xm.xla_device())
+    xla_mat2 = mat2.to(xm.xla_device())
+
+    cpu_res = fn_fallback(M, mat1, mat2)
+    xla_res = dynamo_fn(xla_M, xla_mat1, xla_mat2)
+
```

Comment:
can you use something similar to https://github.com/pytorch/xla/blob/master/torch_xla/core/dynamo_bridge.py#L57 to check there is no fallback ops in this case?

If this works, the fallback should happen on the pytorch cpu side and does not trigger a fallback counter on pytorch/xla end.

We should also check that `xla::cummin` is not in the counter, in case we lower this op in the future.